packages/app-project/pages/_document.js

```diff
@@ -44,7 +44,7 @@ export default class MyDocument extends Document {
 
   render () {
     return (
-      <html>
+      <Html lang="en">
```

Comment:
At some point, this will need to be set dynamically. Maybe wait until after the Next 10 upgrade for that?